daemons/fenced/fenced_remote.c

```diff
@@ -672,6 +672,11 @@ remote_op_timeout_one(gpointer userdata)
     pcmk__set_result(&op->result, CRM_EX_ERROR, PCMK_EXEC_TIMEOUT,
                      "Peer did not return fence result within timeout");
 
+    // The requested delay has been applied for the first device
+    if (op->delay > 0) {
+        op->delay = 0;
+    }
```

Comment:
This change seems sensible to me per review. I haven't figured out how this can be possibly tested though...

Reply:
Maybe add a trace log here just to be able to check for it